controllers/usernamespace/controller.go

```diff
@@ -448,6 +455,50 @@ func (r *CheUserNamespaceReconciler) reconcileProxySettings(ctx context.Context,
 	return err
 }
 
+func (r *CheUserNamespaceReconciler) reconcileIdleSettings(ctx context.Context, targetNs string, checluster *chev2.CheCluster, deployContext *chetypes.DeployContext) error {
+
+	noIdle := pointer.Int32Ptr(-1)
+	if checluster.Spec.DevEnvironments.SecondsOfInactivityBeforeIdling == noIdle && checluster.Spec.DevEnvironments.SecondsOfRunBeforeIdling == noIdle {
```

Comment:
I think we have to compare with `nil` otherwise defaults value `-1` won't be propagated to Config Map.